cmake/EkatMpiUtils.cmake

```diff
@@ -1,20 +1,19 @@
 # Detect the library that provides MPI
 set (EKAT_CMAKE_DIR ${CMAKE_CURRENT_LIST_DIR})
 macro (GetMpiDistributionName DISTRO_NAME)
-  if (CMAKE_CXX_COMPILER AND MPI_CXX_FOUND)
+  if (CMAKE_CXX_COMPILER)
+    find_package (MPI REQUIRED COMPONENTS CXX)
```

Comment:
Do we want `CXX` components for MPI? Have the C++ MPI bindings ever been anything but trouble? I suppose we disable them elsewhere, so maybe this doesn't matter.

Reply:
That's a good point. I will change this to C component.